app/javascript/dashboard/components/widgets/conversation/MessageSignature.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div
+    class="my-0 mx-4 px-1 flex max-h-[8vh] items-baseline justify-between hover:bg-slate-25 dark:hover:bg-slate-800 border border-dashed border-slate-100 dark:border-slate-700 rounded-sm overflow-auto"
+  >
+    <p class="w-fit !m-0">
```

Comment:
What does the `!m-0` do?

Reply:
@scmmishra I added it because the `p` tag has some base styles that will not be removed when we add like this `m-0`. That's the reason I added it as important.

Reply:
Cool, what is the source of the style?

Reply:
@scmmishra Removing the default margin bottom from the paragraph.